### DISCUSSION TITLE
Why is the turbowarp translate issue shut down?

### DISCUSSION THREAD

**Discussion by Dieforrobux** [TurboWarp/translations#24] [Q&A] [answered]
Hello!

3 days ago... I got confused why the Turbowarp Translate issue was shut down? (December 14 maybe some days ago)

We didn't use translate computer.

We didn't ruin anything.

We were trying to discuss

I want to know why it was shut down (thats why its a Q&A)

Link to see what happened and how it was closed and shut down. #1

You can answer comments and i will answer if i can help.

You can help by commenting if you have my issues with this or what i am having currently.

If you got confused my the "#1" Click this one #1 Issue. That is one. Wait! Don't go until i tell you to Now you get the chance to do it. Keep -
scrolling down until you find it got closed. And you can't comment anymore. and then your good and able to explain.

But here is the new one that token over the other one. [Closed github issue](https://github.com/TurboWarp/translations/issues/1)

Thank you for all your help. I will keep on helping turbowarp.

We need your help at transifex. Make your free account with no money here:  [Transifex](https://www.transifex.com/signup/) (With free trial for 14 days)

**Comment by GarboMuffin** [comment, marked as answer]
That issue has been superseded by a new issue, that's all.

**Reply by Dieforrobux** [reply]
Ok. I will just wait for some other answers on the way.

**Reply by Dieforrobux** [reply]
I aleardy have a transfix account (by the way)
